keyboards/handwired/yamsek/readme.md

```diff
@@ -0,0 +1,18 @@
+![Yet another Modular split ergonomic Keybard](https://cdn.hackaday.io/images/8060711492102546379.jpg)
```

Comment:
Also there is a typo here:
```suggestion
![Yet another Modular split ergonomic Keyboard](https://cdn.hackaday.io/images/8060711492102546379.jpg)
```

(And, given the recent news about Imgur, maybe QMK would need to do something about the image hosting requirements… At least this image is apparently just a logo, so losing it won't matter much.)